packages/ttd.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolName = "ttd"
+    $category = "Debuggers"
+
+    # From https://aka.ms/ttd/download
+    $bundleUrl = "https://windbg.download.prss.microsoft.com/dbazure/prod/1-11-319-0/TTD.msixbundle"
+    $bundleSha256 = "f7b80731c3a6994b3763c4100073b101965327d6556fa4bfb553d70ce49be366"
+    $bundlePath = Join-Path ${Env:TEMP} "$($toolName)_1-11-319-0.msixbundle"
+    Get-ChocolateyWebFile -PackageName $toolName -FileFullPath $bundlePath -Url $bundleUrl -Checksum $bundleSha256 -ChecksumType "sha256"
```

Comment:
[nitpick] I would prefer to use the `$packageArgs = @{}` syntax for consistency with other packages

Reply:
In #1058 I have done it like below :
```powershell
$packageArgs = @{
    packageName   = ${Env:ChocolateyPackageName}
    url           = $bundleUrl
    checksum      = $bundleSha256
    checksumType  = "sha256"
    fileFullPath  = Join-Path ${Env:TEMP} "$toolName.msixbundle"
}

Get-ChocolateyWebFile @packageArgs
Add-AppxPackage -Path $packageArgs.fileFullPath
```

Reply:
@naacbin Done.